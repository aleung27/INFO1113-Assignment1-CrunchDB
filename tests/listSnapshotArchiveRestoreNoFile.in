LIST SNAPSHOTS
SET k 8
SET f 2
SNAPSHOT
LIST SNAPSHOTS
RESTORE listSnapshotArchiveRestoreNoFileInput.txt
LIST SNAPSHOTS
SET m 32
SNAPSHOT
LIST SNAPSHOTS
ARCHIVE 1 listSnapshotArchiveRestoreNoFile.txt
