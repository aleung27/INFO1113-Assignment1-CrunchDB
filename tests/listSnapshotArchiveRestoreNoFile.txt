k|8
f|2
