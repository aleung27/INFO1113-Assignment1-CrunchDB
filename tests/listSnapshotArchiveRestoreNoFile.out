> no snapshots

> ok

> ok

> saved as snapshot 1

> 1

> file not found

> 1

> ok

> saved as snapshot 2

> 2
1

> ok

